src/sentry/integrations/jira/integration.py

```diff
@@ -712,6 +712,8 @@ def get_create_issue_config(self, group, user, **kwargs):
                 # allowedValues for some reason doesn't pass enough info.
                 field["choices"] = self.make_choices(client.get_priorities())
                 field["default"] = defaults.get("priority", "")
+                # Don't restore the prior priority choices upon re-rendering b/c they differ by project.
```

Comment:
It's fine to combine backend and frontend changes here. This will be ignored by the frontend, and vice versa for backend.